cloudbuild.yaml

```diff
@@ -62,21 +70,26 @@ steps:
     docker tag bazel/base:static_root_arm64_debian9     gcr.io/$PROJECT_ID/static:latest-arm64
     docker tag bazel/base:static_nonroot_arm64_debian9  gcr.io/$PROJECT_ID/static:nonroot-arm64
 
-    docker tag bazel/base:base_debian9            gcr.io/$PROJECT_ID/base:${COMMIT_SHA}
-    docker tag bazel/base:base_debian9            gcr.io/$PROJECT_ID/base:latest
-    docker tag bazel/base:base_debian9            gcr.io/$PROJECT_ID/base-debian9:${COMMIT_SHA}
-    docker tag bazel/base:base_debian9            gcr.io/$PROJECT_ID/base-debian9:latest
-    docker tag bazel/base:base_debian10           gcr.io/$PROJECT_ID/base-debian10:${COMMIT_SHA}
-    docker tag bazel/base:base_debian10           gcr.io/$PROJECT_ID/base-debian10:latest
-    docker tag bazel/base:base_nonroot_debian9    gcr.io/$PROJECT_ID/base:nonroot
-    docker tag bazel/base:base_nonroot_debian9    gcr.io/$PROJECT_ID/base-debian9:nonroot
-    docker tag bazel/base:base_nonroot_debian10   gcr.io/$PROJECT_ID/base-debian10:nonroot
-    docker tag bazel/base:debug_debian9           gcr.io/$PROJECT_ID/base:debug
-    docker tag bazel/base:debug_debian9           gcr.io/$PROJECT_ID/base-debian9:debug
-    docker tag bazel/base:debug_debian10          gcr.io/$PROJECT_ID/base-debian10:debug
-    docker tag bazel/base:debug_nonroot_debian9   gcr.io/$PROJECT_ID/base:debug-nonroot
-    docker tag bazel/base:debug_nonroot_debian9   gcr.io/$PROJECT_ID/base-debian9:debug-nonroot
-    docker tag bazel/base:debug_nonroot_debian10  gcr.io/$PROJECT_ID/base-debian10:debug-nonroot
+    docker tag bazel/base:base_root_amd64_debian9        gcr.io/$PROJECT_ID/base:${COMMIT_SHA}
+    docker tag bazel/base:base_root_amd64_debian9        gcr.io/$PROJECT_ID/base:latest-amd64
+    docker tag bazel/base:base_root_amd64_debian9        gcr.io/$PROJECT_ID/base-debian9:${COMMIT_SHA}
+    docker tag bazel/base:base_root_amd64_debian9        gcr.io/$PROJECT_ID/base-debian9:latest
```

Comment:
Ideally and eventually, I think this should be a manifest list. We've maintained that `base` and `base-debian9` are basically identical. (And same for `-debian10`.) But I guess you're adding manifest lists only to `base` as a start?

Reply:
Yeah, I wanted to plumb through the main things for each first.  I think we need a bigger discussion around how we (likely) script these variations to cut down on the boilerplate (and sorts of errors I had above)